plone/batching/interfaces.py

```diff
@@ -17,15 +17,15 @@ class IBatch(Interface):
     )
 
     multiple_pages = zope.schema.Bool(
-        title="Boolean indicating wheter there are multiple pages or not"
+        title="Boolean indicating whether there are multiple pages or not"
```

Comment:
@erral If this PR is merged, we may need to update `plone.app.locales` to fix the spelling in the po files and avoid creating fuzzy markers.